core/src/main/scala/cats/data/NonEmptyMapImpl.scala

```diff
@@ -78,6 +78,28 @@ sealed class NonEmptyMapOps[K, A](val value: NonEmptyMap[K, A]) {
   def map[B](f: A => B): NonEmptyMap[K, B] =
     NonEmptyMapImpl.create(Functor[SortedMap[K, *]].map(toSortedMap)(f))
 
+  /**
+   * Applies f to all the keys leaving elements unchanged.
+   */
+  def leftMap[L](f: K => L)(implicit orderL: Order[L]): NonEmptyMap[L, A] = {
+    implicit val orderingL: Ordering[L] = orderL.toOrdering
+    NonEmptyMapImpl.create(toSortedMap.map(Bifunctor[Tuple2].leftMap(_)(f)))
+  }
+
+  /**
+   * Applies f to both keys and elements simultaneously.
+   */
+  def fullMap[L, B](f: (K, A) => (L, B))(implicit orderL: Order[L]): NonEmptyMap[L, B] = {
```

Comment:
I wonder if mapBoth might be a better name here (which is used in WriterT in related way).

Reply:
done